com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/Raytracing/Shaders/Reflections/RaytracingReflections.compute

```diff
@@ -26,13 +28,60 @@
 // Tile size of this compute
 #define RAYTRACING_REFLECTIONS_TILE_SIZE 8
 
+// Input data
 TEXTURE2D_X(_DepthTexture);
-RW_TEXTURE2D_X(float4, _RaytracingDirectionBuffer);
-TEXTURE2D_X(_SsrClearCoatMaskTexture);
-
-// Flag value that defines if a given pixel recieves reflections or not
 TEXTURE2D_X_UINT2(_StencilTexture);
+// Flag value that defines if a given pixel recieves reflections or not
 int _SsrStencilBit;
+TEXTURE2D_X(_SsrClearCoatMaskTexture);
+
+// Output data
+RW_TEXTURE2D_X(float4, _RaytracingDirectionBuffer);
+
+// Structure that holds everything that defines a GGX sample
+struct GGXSample
+{
+    float3 direction;
+    float pdf;
+};
+
+GGXSample GenerateGGXSampleDirection(uint2 currentCoord, float3 normalWS, float3 viewWS, float roughness, int frameIndex)
+{
+    // Create the local ortho basis
```

Comment:
is this function really specialise to this files? Shouldn't this be share.

Reply:
This exact version is only used for perf rtr